app/models/event_collection.rb

```diff
@@ -10,6 +10,32 @@ def any?
   end
 
   def serialize!(scope = {})
-    Events::ArraySerializer.new(self, scope: scope).as_json
+    Events::ArraySerializer.new(self, scope: default_scope.merge(scope)).as_json
+  end
+
+  private
+
+  def default_scope
+    { cache: { likers: likers, attachments: attachments, mentions: mentions } }
+  end
+
+  def likers
+    @likers ||= User.joins(:comment_votes)
+                    .select('users.*').select('comment_votes.comment_id')
+                    .where('comment_votes.comment_id': event_comment_ids)
+                    .group_by(&:comment_id)
+  end
+
+  def mentions
+    @mentions ||= Comment.find(event_comment_ids).map { |c| [c.id, c.mentioned_usernames] }.to_h
```

Comment:
I believe this will allow us, in another iteration, to be smart about who we highlighting as mentions (ie, only highlight members of the group, or people who have actually been notified of the mention)